specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -925,32 +967,35 @@ The JDBC 4.2 specification is available at
 _https://jcp.org/en/jsr/detail?id=221_ .
 
 [[a2553]]
-===== Jakarta API for XML Web Services (JAX-WS) Requirements
+===== Jakarta XML Web Services (JAX-WS™) Requirements (Optional)
 
-The JAX-WS specification provides support for
-web services that use the JAXB API for binding XML data to Java objects.
-The JAX-WS specification defines client APIs for accessing web services
+The Jakarta XML Web Services specification provides support for
+web services that use the Jakarta XML Binding API for binding XML data to Java objects.
+The XML Web Services specification defines client APIs for accessing web services
 as well as techniques for implementing web service endpoints. The Web
 Services for Jakarta EE specification describes the deployment of
-JAX-WS-based services and clients. The Enterprise Beans and Servlet specifications
+XML Web Services-based services and clients. The Enterprise Beans and Servlet specifications
 also describe aspects of such deployment. It must be possible to deploy
-JAX-WS-based applications using any of these deployment models.
+XML Web Services-based applications using any of these deployment models.
 
-The JAX-WS specification describes the
+The Jakarta XML Web Services specification describes the
 support for message handlers that can process message requests and
 responses. In general, these message handlers execute in the same
 container and with the same privileges and execution context as the
-JAX-WS client or endpoint component with which they are associated.
+Web Services client or endpoint component with which they are associated.
 These message handlers have access to the same JNDI _java:comp/env_
 namespace as their associated component. Custom serializers and
 deserializers, if supported, are treated in the same way as message
 handlers.
 
-The JAX-WS specification is available at
-_http://jcp.org/en/jsr/summary?id=224_ .
+The Jakarta XML Web Services specification is available at
+_https://jakarta.ee/specifications/xml-web-services/_ .
 
 ===== Java IDL (Proposed Optional)
 
+*TODO Note:* Since the CORBA module didn't get picked up for Jakarta EE 9, do we remove
```

Comment:
We had voted to remove CORBA and that would remove this section.

Reply:
Thanks for the confirmation!